Changes

```diff
@@ -22,6 +22,18 @@ Working version
 
 ### Runtime system:
 
+* #11865, #11868, #?: Clarify that the operations of a custom block
+  must never access the OCaml runtime. The previous documentation only
+  mentioned the main illicit usages. In particular, since OCaml 5.0,
+  it is no longer safe to call `caml_remove_global_root` or
+  `caml_remove_generational_global_root` from within the C finalizer
+  of a custom block, or within the finalization function passed to
+  `alloc_small_final`. As a workaround, such a finalization operation
```

Comment:
what is `alloc_small_final`? `git grep` does not return any result

Reply:
I meant `caml_alloc_final`; pushing an update shortly.